app/models.py

```diff
@@ -468,6 +474,16 @@ def serialize_for_list(self):
         }
 
 
+class OrganisationEmailBranding(db.Model):
```

Comment:
Hi David, thanks for the suggestion! We discussed with Gozie, and decided we will rename the new class to `OrganisationEmailBrandingPool`, and leave column names and table name in Postgres as they are.

Reply:
> One note of caution with your suggestion. OrganisationEmailBrandingPool is a model representing a single row in the database table.

Hi David, I don't think this is quite right. One pool is spread across possibly many rows - one row is a connection between an organisation and one branding, but a pool is a relation between one organisation and many brandings, so it'a possibly many rows, selected on organisation id.

We originally named this model `OrganisationEmailBranding`, because for a similar table, `user_to_service`, model name is `UserService`, so wanted to be consistent with that. Maybe we could call it 'EmailBrandingToOrganisation' instead?

Reply:
After some thought I agree with you best option is option number 2:

> Rename the attribute email_branding to default_email_branding on the Organisastion model class. You'd probably want to do the same for letter_branding to default_letter_branding as well for consistency.

I will add first steps for that change to this PR.

Reply:
I also agree with going with option number 2. I have made all the other requested changes made so that the only thing left to do is to carry out the suggested changes in option 2.

Reply:
Maybe we go pragmatic. Nothing will ever be perfect. Here is a suggestion for something that is fairly easy and quick to implement.

Add an inline comment to the `Organisation` class `email_branding` and `letter_branding` attributes saying these are the default brandings and not to be confused with the email branding pool. Then we can keep the class `OrganisationEmailBranding` name as it currently is. This keeps it consistent with `UserService` for example.

Thoughts?

Reply:
Sounds good, I added the comments. I didn't say anything about letter branding pool though as we don't have one yet, so that could be confusing.